lib/ipmi_sensor.c

```diff
@@ -260,6 +260,183 @@ print_thresh_setting(struct sdr_record_full_sensor *full,
 	}
 }
 
+static void
+dump_sensor_fc_thredshold_csv(int thresh_available,
+			 const char *thresh_status,
```

Comment:
These lines are strangely aligned with a mix of tabs and spaces. Please use spaces only and align to `int` above.
Alternatively, put a line break after `(` and simply indent all arguments with a single TAB.